html_test_harness/README.md

```diff
@@ -0,0 +1,8 @@
+# HTML Test Harness
+
+This is a temporary set of utilities to allow testing HTML rewriting.
+
+## Interesting things
+
+ * `data/example_urls.csv` - A spreadsheet of URLs and data
+ * `bin/scan.py` - Will request and update a local cache of data about the URLs
```

Comment:
Yeah, this isn't likely to stick around for long, so I don't want to put too much effort in. Also this isn't actually the ticket to produce the harness.